Source/WebCore/Modules/modern-media-controls/controls/background-click-delegate-notifier.js

```diff
@@ -37,6 +37,7 @@ class BackgroundClickDelegateNotifier
 
     handleEvent(event)
     {
+        event.stopPropagation();
```

Comment:
If this fix is specific to the `in-window` mode, should it not be guarded by a condition? Would this not prevent propagation in inline mode with custom controls as well?

Reply:
You're right - that's what's causing the layout test failures I believe

Reply:
Fixed in new upload